src/Illustration/index.js

```diff
@@ -7,8 +7,27 @@ import { SIZE_OPTIONS, baseURL } from "./consts";
 
 import type { Props } from "./index";
 
-export const StyledImage = styled.img`
-  height: ${({ tokens, size }) => tokens.height[size]};
+const getHeightToken = ({ theme, size }) => {
+  const tokens = {
+    [SIZE_OPTIONS.SMALL]: theme.orbit.heightIllustrationSmall,
+    [SIZE_OPTIONS.MEDIUM]: theme.orbit.heightIllustrationMedium,
+  };
+  return tokens[size];
+};
+
+const getURL = (retina = false) => ({ theme, size, name }) => {
+  const height = parseInt(getHeightToken({ theme, size }), 10);
+  return retina
+    ? `${baseURL}/illustrations/0x${height * 2}/${name}.png 2x`
+    : `${baseURL}/illustrations/0x${height}/${name}.png`;
+};
+
+export const StyledImage = styled.img.attrs({
+  name: false, // Prevent rendering name in DOM
```

Comment:
Maybe we can use this:
```jsx
export const StyledImage = styled(props => (
  <img
    src={getURL()(props)}
    srcSet={getURL(true)(props)}
    className={props.className}
    data-test={props.dataTest}
    alt={props.name}
  />
))
```
The name is still rendering to the DOM and I would stick to one solution - either `attrs` or custom arrow fuction.

Reply:
Or if you wanna stick to this solution, use `name: undefined`